linux/LinuxProcessList.c

```diff
@@ -1151,12 +1150,18 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, const char*
       if (proc->state == 'R')
          pl->runningTasks++;
       proc->updated = true;
+
+      close(procFd);
+
       continue;
 
       // Exception handler.
 
 errorReadingProcess:
       {
+         if (procFd >= 0)
+            close(procFd);
+
          if (preExisting) {
             ProcessList_remove(pl, proc);
```

Comment:
Shouldn't this trigger the diff highlighting logic?

Reply:
Diff highlighting is fixed now